psycopg3/transaction.py

```diff
@@ -13,35 +13,73 @@
 if TYPE_CHECKING:
     from .connection import Connection
 
+_log = logging.getLogger(__name__)
+
+
+class Rollback(Exception):
+    """
+    Exit the current Transaction context immediately and rollback any changes
+    made within this context.
+
+    If a transaction context is specified in the constructor, rollback
+    enclosing transactions contexts up to and including the one specified.
+    """
+
+    def __init__(self, transaction: "Transaction" = None) -> None:
+        self.transaction = transaction
+
+    def __str__(self) -> str:
```

Comment:
Agree. And I think the same goes for `Transaction.__str__()` too?

Reply:
Yup, I think I dropped a note there too.

Reply:
Would you say `__repr__()` warrants tests?

Since I can see you are quite strict about what is part of the interface (and therefore should have test coverage) and what isn't.

Reply:
I would be ok without tests for repr. Let's just assume it won't explode.

> Also, I think `__qualname__` is the new canonical replacement for `__name__`.

If it looks the best, sure